src/data/database.js

```diff
@@ -880,6 +880,328 @@ class Database {
     }
   }
 
+  async createFxQuote (txn, conversionRequestId) {
+    try {
+      const newFxQuote = {
+        conversionRequestId
+      }
+
+      const res = await this.queryBuilder('fxQuote')
+        .transacting(txn)
+        .insert(newFxQuote)
+
+      newFxQuote.fxQuoteId = res[0]
+
+      this.writeLog(`inserted new fxQuote in db: ${util.inspect(newFxQuote)}`)
+      return newFxQuote
+    } catch (err) {
+      this.writeLog(`Error in createFxQuote: ${getStackOrInspect(err)}`)
```

Comment:
I'd suggest stopping using  `this.writeLog` to log error, coz it uses `debug` severity, so we won't have such logs in production